src/sdg/conf/base.py

```diff
@@ -574,3 +574,6 @@
     )
 
 SPECTACULAR_SETTINGS["SERVERS"] = SDG_API_SERVER_INSTANCES
+
+WHITELISTING_ENABLED = config("WHITELISTING_ENABLED", default=True)
+REAL_USER_IP = config("REAL_USER_IP", default="X-Real-IP")
```

Comment:
Liever beschrijven dan de waarde van de header herhalen.

```suggestion
CLIENT_IP_HTTP_HEADER = config("REAL_USER_IP", default="X-Real-IP")
```